src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -364,6 +364,11 @@ private static Lazy<HashSet<string>> CreateInternalsVisibleToMap(IAssemblySymbol
                     }
 
                     var value = (string)typeNameConstant.Value;
+                    if (string.IsNullOrWhiteSpace(value))
+                    {
+                        continue;
+                    }
+
                     var commaIndex = value.IndexOf(',');
                     var assemblyName = commaIndex >= 0 ? value.Substring(0, commaIndex).Trim() : value;
```

Comment:
I understand the desire to do the minimum for the watson bug, but did we examine the customer scenario when empty strings are passed, etc?  I wouldn't want to fix this, and then have it crash again two lines further down if it was an empty or whitespace string.